### ISSUE TITLE
Reset the default config or manually change the config

### ISSUE THREAD

**Issue by rtrembecky** [noah-nuebling/Mac-Mouse-Fix#159]
Hi, is there a way to reset the config back to the defaults? Or a way to change the config manually via editing some file? My problem is that I removed all the key assignments in the app and I'm unable to bring them back - middle-clicking on the plus sign doesn't do anything. However, the original config for the buttons 3/4/5 worked well for me.

console events for clicking all 5 buttons on my mouse (Gigabyte M6900):
```console
default	12:35:28.215176+0200	Mac Mouse Fix Helper	Received CG Button Input - NSEvent: type=LMouseDown loc=(862.885,733.87) time=12618.9 flags=0 win=0x0 winNum=0 ctxt=0x0 evNum=812 click=1 buttonNumber=0 pressure=1 deviceID:0x0 subtype=0
default	12:35:29.599940+0200	Mac Mouse Fix Helper	Received CG Button Input - NSEvent: type=LMouseUp loc=(862.885,733.87) time=12620.3 flags=0 win=0x0 winNum=0 ctxt=0x0 evNum=812 click=1 buttonNumber=0 pressure=0 deviceID:0x0 subtype=0
default	12:35:34.284545+0200	Mac Mouse Fix Helper	Received CG Button Input - NSEvent: type=RMouseDown loc=(851.946,735.541) time=12625.0 flags=0 win=0x0 winNum=0 ctxt=0x0 evNum=813 click=1 buttonNumber=1 pressure=1 deviceID:0x0 subtype=0
default	12:35:34.356679+0200	Mac Mouse Fix Helper	Received CG Button Input - NSEvent: type=RMouseUp loc=(851.946,735.541) time=12625.1 flags=0 win=0x0 winNum=0 ctxt=0x0 evNum=813 click=1 buttonNumber=1 pressure=0 deviceID:0x0 subtype=0
default	12:35:36.183024+0200	Mac Mouse Fix Helper	Received CG Button Input - NSEvent: type=OtherMouseDown loc=(809,734) time=12626.9 flags=0 win=0x0 winNum=0 ctxt=0x0 evNum=98 click=1 buttonNumber=2 pressure=1 deviceID:0x0 subtype=0
default	12:35:36.300932+0200	Mac Mouse Fix Helper	Received CG Button Input - NSEvent: type=OtherMouseUp loc=(809,734) time=12627.0 flags=0 win=0x0 winNum=0 ctxt=0x0 evNum=98 click=1 buttonNumber=2 pressure=0 deviceID:0x0 subtype=0
default	12:35:38.194080+0200	Mac Mouse Fix Helper	Received CG Button Input - NSEvent: type=OtherMouseDown loc=(810,734) time=12628.9 flags=0 win=0x0 winNum=0 ctxt=0x0 evNum=99 click=1 buttonNumber=3 pressure=1 deviceID:0x0 subtype=0
default	12:35:38.272558+0200	Mac Mouse Fix Helper	Received CG Button Input - NSEvent: type=OtherMouseUp loc=(810,734) time=12629.0 flags=0 win=0x0 winNum=0 ctxt=0x0 evNum=99 click=1 buttonNumber=3 pressure=0 deviceID:0x0 subtype=0
default	12:35:39.051887+0200	Mac Mouse Fix Helper	Received CG Button Input - NSEvent: type=OtherMouseDown loc=(810,734) time=12629.8 flags=0 win=0x0 winNum=0 ctxt=0x0 evNum=100 click=1 buttonNumber=4 pressure=1 deviceID:0x0 subtype=0
default	12:35:39.186070+0200	Mac Mouse Fix Helper	Received CG Button Input - NSEvent: type=OtherMouseUp loc=(811,738) time=12629.9 flags=0 win=0x0 winNum=0 ctxt=0x0 evNum=100 click=1 buttonNumber=4 pressure=0 deviceID:0x0 subtype=0
```

I'm on macOS 11.4.

Maybe I'll give a try to SteerMouse as you suggested on other issues, but I think resetting the config would be enough for me.

**Comment by noah-nuebling** [comment]
Hey there @rtrembecky, thanks for your detailed feedback!

You should be able to reveal the config file in Finder by Command(⌘)-Shift(⇧)-Clicking the App Icon in under the "More..." view.

https://user-images.githubusercontent.com/40808343/135691036-b2756b4c-eec7-4ad0-be8e-403b8f9de5aa.mov

(This video is of the 2.0 version, but it should work the same under 1.0 IIRC)

If you delete the config file in Finder and restart Mac Mouse Fix, the default configuration should come back.

Let me know if that helps or if you have any more questions, but please understand that I don't have a lot of time to tend to the project right now.

**Comment by rtrembecky** [comment]
hi, thanks for the help. I tried to be even hackier and I went to the Mac Mouse Fix application content to see the default config:
![image](https://user-images.githubusercontent.com/6044119/135731733-f50ab3c6-0083-4241-8cb6-f1f41029e9dd.png)

for future reference for anybody interested - in Finder, go to Applications, find the app, right-click and choose show contents:
![image](https://user-images.githubusercontent.com/6044119/135731764-ab7dbb19-1eb6-4732-9370-f43c22789b0e.png)

I was able to copy the `Remaps` keys into the `config.plist` and restarted the app. the default remaps indeed showed themselves back 🙂 

so thanks again for the information and have a nice day.
